torchao/utils.py

```diff
@@ -126,6 +126,10 @@ def right_inverse(self, tensor):
         return plain_tensors
 
 def unwrap_tensor_subclass(model, filter_fn=None):
+    """Unwraps (nested) tensor subclass in the model to plain tensors
+    This is a workaround to make a model with tensor subclass to work with `torch.export.export`
```

Comment:
I'm not clear on why this is needed, perhaps link to the github issue in pytorch core so we know when we can remove this in future